### DISCUSSION TITLE
Support of SD XL

### DISCUSSION THREAD

**Discussion by InternalMegaT** [TheLastBen/fast-stable-diffusion#2441] [Q&A]
Does Dreambooth Support SD XL? If not can it?

**Comment by chubbypillow93** [comment]
> Does Dreambooth Support SD XL? If not can it?

I'm waiting for a Dreambooth colab for SDXL as well :( I've tried training LoRA of a person's face on SDXL but the result was quite meh, I saw some very good SDXL face LoRAs on Civitai but they're really really large, it's probably because I'm not familiar with the configs, but Dreambooth trained checkpoint on a fine-tuned SD1.5 base model looked so much better :(

**Comment by THX1139b** [comment]
I think we're supposed to use the Runpod version for Dreambooth training. I myself however has never gotten Fast Stable Diffusion to work on Runpod. Trying it now it simply stops after a couple of cells. No model is downloaded or linked. No photos are uploaded. It simply stops responding.

99% certain it is because I don't understand it. But having reached my point of stupidity I'm hoping for a way to use it like the "old" Dreambooth that was simple enough to get running.
